app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/MapViewModel.java

```diff
@@ -140,6 +146,11 @@ public void mapReady() {
         mDisplayMapEntitiesInteractor = mDisplayMapEntitiesInteractorFactory.create(this);
```

Comment:
It's even worse than that actually,
There's a repository for the viewer's "last saved camera configuration",  and interactors to use it.
We should investigate it further and decide what to with it.
I'll open an issue for that.